spec/components/parameters/files/startTimestamp.ts

```diff
@@ -0,0 +1,16 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../utils/ref';
+
+const startTimestamp: ParameterObject = {
+  name: 'startTimestamp',
+  in: 'query',
+  required: false,
+  description: 'Date the file were created',
```

Comment:
Melhor atualizar a descrição (já que não é mais date), talvez algo como:
`Timestamp after which, the files to be returned, were created`
`Moment after which, the files to be returned, were created`

E o mesmo pro end.